docs/snippets/ov_preprocessing_migration.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/runtime/core.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <openvino/core/preprocess/pre_post_process.hpp>
+
+#include "inference_engine.hpp"
+
+int main_new() {
+    std::string model_path;
+
+    ov::Core core;
+    std::shared_ptr<ov::Model> model = core.read_model(model_path);
+    ov::preprocess::PrePostProcessor ppp(model);
+
+    {
+    //! [ov_mean_scale]
+ov::preprocess::PrePostProcessor ppp(model);
+ov::preprocess::InputInfo& input = ppp.input("tensor_name");
+// we only need to know where is C dimension
+input.model().set_layout("???C");
+// specify scale and mean values, order of operations is important
+input.preprocess().mean(0.1f).scale({ 0.5f, 0.05f, 0.01f });
```

Comment:
suggesting to get example values from some real models. There is usually means are ~128, and scale range is ~50-128

Reply:
done